generators/app/templates/terraform/bitops.config.yaml

```diff
@@ -1,6 +1,4 @@
 terraform:
     cli: {}
     options:
-        command: apply
-        version: "0.13.2"
-        fetch-kubeconfig: false
\ No newline at end of file
+        stack-action: apply
```

Comment:
A note about moving `stack-action` under the `cli` per https://github.com/bitops-plugins/terraform/pull/17

Alternatively, we could just remove the option here, - that'll switch to `plan` by default.
@PhillypHenning thoughts?

Reply:
Updated in https://github.com/bitovi/bitops-operations-repo-generator/pull/2/commits/e772b4ef79a8a87f36cdce4bfb341e5d9ac593c0 following your Terraform schema changes in `v2.2.0`